### PR TITLE
fix: handle the file not found error gracefully

### PR DESCRIPTION
**Pull request by golesuman**
## Description

Give meaningful error message than giving the full stack trace when the file is not found.

## Related Issue

Fixes: #44 

## Type of Change

Please mark the appropriate option below to describe the type of change your pull request introduces:

-   [x] Bug fix
-   [ ] New feature
-   [ ] Enhancement
-   [ ] Documentation update
-   [ ] Refactor
-   [ ] Other (please specify)

## Checklist

-   [x] My pull request has a clear title and description.
-   [x] I have used [semantic commit messages](https://seesparkbox.com/foundry/semantic_commit_messages).
        Examples: `"fix: Fixed foobar bug"`, `"feat(accounts): Added foobar feature"`.
-   [ ] I have added/updated the necessary documentation on `README.md`.
-   [x] I have added appropriate test cases (if applicable) to ensure the changes are functioning correctly.

## Additional Notes

[Add any additional notes or context that you think the reviewers should know about.]

By submitting this pull request, I confirm that I have read and complied with the contribution guidelines of this project.

### CODE REVIEW

**Comment by aj3sh** [comment]
Could you please add a test for this case?

**Comment by golesuman** [comment]
> Could you please add a test for this case?

yeah done please check @aj3sh

**Review by aj3sh — APPROVED** [review]
Looks Good.

File: src/commitlint/cli.py [hunk 1]
```diff
@@ -228,6 +228,10 @@ def main() -> None:
         console.error(f"{ex}")
         sys.exit(1)
 
+    except FileNotFoundError as ex:
+        console.error(f"{ex}")
```
Comment (aj3sh) [inline src/commitlint/cli.py:232]: query: What will be the output of this error message?

Comment (aj3sh) [inline src/commitlint/cli.py:232]: Please provide a user-friendly error. The current output is something like `"[Errno 2] No such file or directory: 'path/to/file'"`.

**Review by aj3sh — CHANGES_REQUESTED** [review]

**Review by aj3sh — APPROVED** [review]

**Pull request merged**